Makefile

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+all: build
+
+build:
```

Comment:
Please call this `osbs-build` as I can see us using this Makefile for other purposes too.